thirdparty/beatroot/CMakeLists.txt

```diff
@@ -16,3 +16,4 @@ set(MODULE_SRC
 setup_module()
 
 target_no_warning(${MODULE} -Wno-conversion)
+target_no_warning(${MODULE} -Wno-float-equal)
```

Comment:
This doesn't seem to help for some bizarre reason?!?

Reply:
Ah, this is because it gets `#include`'ed by src/importexport/midi/internal/midiimport/importmidi_beat.cpp